.gitignore

```diff
@@ -1,2 +1,5 @@
 __pycache__
 *.sqlite3
+
+# mac specific
+*.DS_Store
```

Comment:
These are just folder organization files that mean nothing to a project but you can't really delete them from macs...